projects/ngx-avatar/src/lib/sources/gravatar.ts

```diff
@@ -1,9 +1,22 @@
-import isRetina from 'is-retina';
```

Comment:
Why dropping this exception? It seems so simple.

Reply:
Because it does not have typings available, and like you said the code is extremely simple, has not changed for the [past 6 years](https://github.com/KyleAMathews/isRetina/commit/5e877c15d7d6b328103a9960113093712cb5e762), has never had a [single issue or PR](https://github.com/KyleAMathews/isRetina/issues?q=). So it is trivial to copy the code into our project directly, and never have any maintenance cost anymore.